packages/scandipwa/src/component/SearchItem/SearchItem.style.scss

```diff
@@ -10,18 +10,23 @@
  */
 
 :root {
+    --search-item-hover-color: #eee;
     --search-item-divider-color: #eee;
     --search-item-title-color: #000;
 }
 
 .SearchItem {
-    padding-inline-start: 0;
-    margin: 24px 0;
+    padding-inline-start: 20px;
+    padding: 12px;
```

Comment:
So you making padding left to be 20px and then overriding it to be 12px. 
I guess out here something wrong.